sdk/master_board_sdk/license.txt

```diff
@@ -0,0 +1,29 @@
+BSD 3-Clause License
+
+Copyright (c) 2019, New York University and Max Planck Gesellschaft.
```

Comment:
Shouldn't LAAS-CNRS be mentioned ?

Reply:
Arg! sorry my bad!
I update this

Reply:
Acutally there is a license file already at the root of the master-board so I will delete this one

Reply:
What is `solo-bl` in https://github.com/open-dynamic-robot-initiative/master-board/blob/master/LICENSE#L3 ?